cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -269,7 +280,7 @@ phases:
               set -v
               echo "Calling chef-client with /etc/parallelcluster/image_dna.json"
               cat /etc/parallelcluster/image_dna.json
-              cinc-client --local-mode --config /etc/chef/client.rb --log_level info --force-formatter --no-color --chef-zero-port 8889 --json-attributes /etc/parallelcluster/image_dna.json --override-runlist aws-parallelcluster-entrypoints::install
+              cinc-client --local-mode --config /etc/chef/client.rb --log_level debug --force-formatter --no-color --chef-zero-port 8889 --json-attributes /etc/parallelcluster/image_dna.json --override-runlist aws-parallelcluster-entrypoints::install
```

Comment:
Why debug?

Reply:
I had it on debug for ADC reasons but I have now switched it back to info